internal/client/client.go

```diff
@@ -237,11 +237,10 @@ func (a *connArray) Close() {
 		a.batchConn.Close()
 	}
 
-	for i, c := range a.v {
+	for _, c := range a.v {
```

Comment:
- if `c` can never be `nil`, we can remove following `if`
- will it cause problem if someone call `Close` multiple times?

Reply:
- if `c` can never be `nil`, we can remove following `if`
Of course.

- will it cause problem if someone call `Close` multiple times?
`connArray.Close()` isn't able to be called multiple times even without this change, and we protect it by lock and it can't be called mutilple times.

Reply:
you are right.